src/ServiceControl.Monitoring/Infrastructure/HistoryPeriod.cs

```diff
@@ -21,7 +22,7 @@ static HistoryPeriod()
         HistoryPeriod(TimeSpan value)
         {
             Value = value;
-            NumberOfIntervals = 20; //Number of points on the graph
+            NumberOfIntervals = 40; //Number of points on the graph
```

Comment:
This doubles the memory needed for storing metrics, right? Any measurements were taken?

Reply:
@tmasternak ☝️

Reply:
You mean tests that would validate that memory consumption changes as expected?

Reply:
I mean any test, even doing a smoke test. This looks like a not so small change in terms of memory consumption. Or maybe, it's nothing that big?

Reply:
@Scooletz I've updated issue description with memory consumption tests. IMO the memory consumption growth of 19 MB is something we can live with.

I'm more concerned with increased times for data queries. The mean went over 50 ms and is around 89 ms on the build machine.

Reply:
@Scooletz after thinking a bit more on the number of intervals I've realized that we need more only for the smallest interval (1 min). 

I've made the intervals per history period 20 for any period larger than 1 min. Hopefully that won't break the perf tests we have. 

I propose to keep the memory measurements in the description for future reference.

Reply:
Interesting finding. Still 89ms for a service accessed by a single user without too long network hops seems ok-ish for a reporting tool. 

Nice findings!